src/org/org.ts

```diff
@@ -873,8 +873,13 @@ export class Org extends AsyncOptionalCreatable<Org.Options> {
 
   /**
    * Returns the JSForce connection for the org.
+   *
+   * @param apiVersion The API version to use for the connection.
    */
-  public getConnection(): Connection {
+  public getConnection(apiVersion?: string): Connection {
```

Comment:
@mshanemc after noodling on this, my only reservation is the implicit permanent state change to the original connection. Maybe deep clone the connection, set the api version and return the clone.

Reply:
Pete and agreed to make this clearer via jsdoc instead.